js-exercises/better-string-lib/betterStringLib.js

```diff
@@ -0,0 +1,31 @@
+
+
+const equal = (str1, str2) => {
+    if (typeof str1 !== 'string' || typeof str2 !== 'string') {
+        throw TypeError('Parameters should be of type String');
+    }
+    return str1.normalize() === str2.normalize();
+}
+
+const reverse = (str) => {
+    if (typeof str != 'string') {
+        throw TypeError('Parameter should be of type String');
+    }
+
+    let charCodeArray = [];
```

Comment:
I think better name is - codePointArray something represents this is codePoint .codepoint and charCode are different.